chapter02/qli/src/main/java/spring/training/model/User.java

```diff
@@ -0,0 +1,23 @@
+package spring.training.model;
+
+public class User {
```

Comment:
I prefer to create a new object for the password change :) What happens if this object is in a Map(assuming you implemented the handshake between `equals` and `hashcode` 
Mutable objects are hidden bugs in my view